docs/book/src/developer/architecture/controllers/control-plane.md

```diff
@@ -229,19 +229,25 @@ The `status` object **may** define several fields:
 
 ## Example usage
 
-``` yaml
-kind: KubeadmControlPlane
-apiVersion: cluster.x-k8s.io/v1alpha3
-metadata:
-  name: kcp-1
-  namespace: default
-spec:
-  infrastructureTemplate:
-    name: kcp-infra-template
-    namespace: default
-  kubeadmConfigSpec:
-    clusterConfiguration:
-  version: v1.16.2
+```yaml
+apiVersion: v1
+items:
+  - apiVersion: controlplane.cluster.x-k8s.io/v1beta1
+    kind: KubeadmControlPlane
+    metadata:
+      name: kcp-1
+      namespace: default
+    spec:
+      kubeadmConfigSpec:
+        clusterConfiguration:
```

Comment:
Why is this key empty, is that valid yaml?

Reply:
I think it's not valid yaml - it's like this because I didn't want to change anything unnecessarily from the previous example unless there was some reason. I think this is really an example structure so it doesn't have to be valid yaml or anything like that.

I've removed this:
```yaml
       kubeadmConfigSpec:
        clusterConfiguration: 
```
completely now as it's not part of the contract so I don't think it adds anything to the expected shape of a control plane object.